doc/changelog.rst

```diff
@@ -3,8 +3,156 @@
+-  Add height for sfcWindmax in MPI HighRes models (:pull:`2292`) by :user:`malininae`
+-  Fixed 'positive' attribute in custom rtnt table (:pull:`2367`) by :user:`schlunma`
+-  Fix `positive` attributes in custom CMOR variables (:pull:`2380`) by :user:`schlunma`
+-  Log CMOR check and generic fix output to separate file (:pull:`2361`) by :user:`schlunma`
+
+Computational performance improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  More lazy fixes and preprocessing functions (:pull:`2325`) by :user:`bouweandela`
+-  Made preprocessors mask_landsea, mask_landseaice and mask_glaciated lazy  (:pull:`2268`) by :user:`joergbenke`
+-  Use lazy weights for `climate_statistics` and `axis_statistics` (:pull:`2346`) by :user:`schlunma`
+-  Fixed potential memory leak in `local_solar_time` (:pull:`2356`) by :user:`schlunma`
+-  Implement lazy area weights (:pull:`2354`) by :user:`schlunma`
+-  Avoid large chunks in `climate_statistics` preprocessor function with `period='full'` (:pull:`2404`) by :user:`bouweandela`
+
+Documentation
+~~~~~~~~~~~~~
+
+-  Add highlights to the v2.10 changelog (:pull:`2282`) by :user:`bouweandela`
+-  Use short links in changelog (:pull:`2287`) by :user:`bouweandela`
+-  National Computing Infrastructure (NCI), Site specific configuration (:pull:`2281`) by :user:`rbeucher`
+-  Update `multi_model_statistics` doc with latest changes (new operators, etc.) (:pull:`2321`) by :user:`schlunma`
+-  Fix Codacy badge (:pull:`2382`) by :user:`bouweandela`
+-  Change 'mean' to 'percentile' in doc strings of preprocessor statistics (:pull:`2327`) by :user:`lukruh`
+-  Fixed typo in doc about weighted statistics (:pull:`2387`) by :user:`schlunma`
+
+Fixes for datasets
+~~~~~~~~~~~~~~~~~~
+
+-  Added fix for wrong units of `clt` for CIESM and FIO-ESM-2-0 (:pull:`2330`) by :user:`schlunma`
+-  Cmip6 gfdl_cm4: fix tas height fix to work for concatenated scenarios (:pull:`2332`) by :user:`mwjury`
+-  Cordex GERICS REMO2015  lon differences above 10e-4 (:pull:`2334`) by :user:`mwjury`
+-  Download ICON grid without locking (:pull:`2359`) by :user:`bouweandela`
+-  Added ICON fixes for hfls and hfss (:pull:`2360`) by :user:`diegokam`
+-  Added ICON fix for rtnt (:pull:`2366`) by :user:`diegokam`
+-  Expanded ICON extra facets (:pull:`2379`) by :user:`schlunma`
+-  Add 10m height coordinate to SfcWind GFDL-CM4 instead of 2m height (:pull:`2385`) by :user:`Karen-A-Garcia`
+-  Cordex wrf381p: fix tas,tasmax,tasmin height (:pull:`2333`) by :user:`mwjury`
+-  Several minor fixes needed for marine BGC data. (:pull:`2110`) by :user:`ledm`
+
+Installation
+~~~~~~~~~~~~
+
+-  Updated iris pin to `iris>=3.6.1` (:pull:`2286`) by :user:`schlunma`
+-  Pin pandas yet again avoid new `2.2.1` as well (:pull:`2353`) by :user:`valeriupredoi`
+-  Pin esmpy <8.6.0 (:pull:`2402`) by :user:`valeriupredoi`
+-  Update Iris pin to avoid using versions with memory issues (:pull:`2408`) by :user:`chrisbillowsMO`
+
+Iris
```

Comment:
I would remove this section and move the pull requests to other sections, as the title Iris does not convey much information for readers of the changelog.

Reply:
We're thinking of moving all the Iris section PRs to "enhancements" as they all deal with underlying changes in Iris since the last release.  Does that seem reasonable?